scripts: hid_configurator: Update docs how to install hidapi DLL (#12216)
nrfconnect/sdk-nrf: scripts/hid_configurator/README.rst

1. Download the HIDAPI library from `HIDAPI releases`_.
   Use the bundled DLL or build it according to instructions in `HIDAPI library`_.

>    To install the DLL, select the appropriate HIDADPI library version for the used Windows system (either ``x86`` or ``x64``).

Reviewer: Maybe we can make it a separate step ?

>    Next, paste :file:`hidapi.dll` and `hidapi.lib` files into either:

Reviewer: ```suggestion
Next copy the :file:`hidapi.dll` file and paste it into either:
```

Reviewer: I think we do not need lib file at all here

Author: Yea, DLL only should be sufficient

>    * The ``C:\Windows\System32`` directory.

Reviewer: ```suggestion
   * The :file:`Windows\System32` directory, for example :file:`C:\Windows\System32`.
```

>    * The directory where used Python executable is located.

Reviewer: Consider reordering it (the second option should be actually preferred)

1. Download the HIDAPI library from `HIDAPI releases`_.
   Use the bundled DLL or build it according to instructions in `HIDAPI library`_.

#. Select the appropriate HIDADPI library version for the used Windows system (either ``x86`` or ``x64``).

>    Copy the :file:`hidapi.dll` file and paste it into either:

Reviewer 2: ```suggestion
   Copy the :file:`hidapi.dll` file and paste it into either of the following directory:
```
nit

> #. To install the DLL, select the appropriate HIDADPI library version for the used Windows system (either ``x86`` or ``x64``).
>    Next, copy the :file:`hidapi.dll` file and paste it into either:

Reviewer: ```suggestion
#. Select the appropriate HIDADPI library version for the used Windows system (either ``x86`` or ``x64``).
    Copy the :file:`hidapi.dll` file and paste it into either:
```

   * The directory where used Python executable is located.

>    * The :file:`Windows\System32` directory, for example :file:`C:\Windows\System32`.

Reviewer 2: ```suggestion
   * The :file:`Windows/System32` directory, for example :file:`C:\\Windows\\System32`.
```

>    * The :file:`Windows/System32` directory, for example :file:`C:\\Windows\\System32`.

Reviewer 2: ```suggestion
   * The :file:`Windows\\System32` directory, for example :file:`C:\\Windows\\System32`.
```
We had to change here also, since this is also a windows path. missed in my initial review.
